src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem-Container.js

```diff
@@ -1,10 +1,17 @@
 import { connect } from 'react-redux';
 import RequestListItem from './RequestListItem';
+import { requestInboxAlert } from '../../../../Actions/requestInboxAlertActions';
 
-const mapStateToProps = () => {
+const mapStateToProps = state => {
```

Comment:
instead of state just pass the `requestInboxAlert`

Reply:
yep! using destructuring